packages/cli/src/commands/generate/service/__tests__/fixtures/multiWordServiceCrud.ts

```diff
@@ -0,0 +1,45 @@
+import { db } from 'src/lib/db'
+import {
+  UserProfileWhereUniqueInput,
+  UserProfileCreateInput,
+  UserProfileUpdateInput,
+} from '@prisma/client'
+
+
+export const userProfiles = () => {
+  return db.userProfile.findMany()
+}
+
+export const userProfile = ({ id }: UserProfileWhereUniqueInput) => {
+  return db.userProfile.findOne({
+    where: { id },
+  })
+}
```

Comment:
@guledali updated generated files to include types, let me know what you think

Reply:
LGTM